guest-tools/td_virsh_tool.sh

```diff
@@ -59,6 +59,7 @@ Available options:
 -h,   --help            Print this help and exit
 -c D, --clean D         Clean domain with name D (use "all" to clean all)
 -n N, --n_instances N   Launch N instances
+-l, --list              List VMs
```

Comment:
Could you align `--list` with `--n_instances` in line above to make the formatting nicer?